src/pages/form-validation/fixed-length-passcode/index.tsx

```diff
@@ -0,0 +1,215 @@
+import React, { useCallback, useState } from 'react';
+import {
+    AppBar,
+    Button,
+    CircularProgress,
+    Divider,
+    Hidden,
+    IconButton,
+    InputAdornment,
+    InputProps,
+    TextField,
+    Toolbar,
+    Typography,
+} from '@material-ui/core';
+import { Done, Menu, Refresh } from '@material-ui/icons';
+import { makeStyles, Theme, useTheme } from '@material-ui/core/styles';
+import { useDispatch } from 'react-redux';
+import { TOGGLE_DRAWER } from '../../../redux/actions';
+import * as Colors from '@pxblue/colors';
+
+type OnChangeHandler = InputProps['onChange'];
+
+const useStyles = makeStyles((theme: Theme) => ({
+    appbarRoot: {
+        padding: 0,
+    },
+    toolbarGutters: {
+        padding: '0 16px',
+    },
+    containerWrapper: {
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'start',
+        flex: '1 1 0',
+        backgroundColor: 'white',
+        height: 'calc(100vh - 64px)',
+    },
+    container: {
+        width: '100%',
+        maxWidth: 480,
+        margin: theme.spacing(4),
+        [theme.breakpoints.down('xs')]: {
+            margin: theme.spacing(2),
+            maxWidth: '100%',
+        },
+    },
+    submitButton: {
+        marginTop: theme.spacing(4),
+        [theme.breakpoints.down('xs')]: {
+            width: '100%',
+        },
+    },
+    details: {
+        marginTop: theme.spacing(1)
+    }
+}));
+
+export const FixedLengthPasscodeValidation = (): JSX.Element => {
+    const theme = useTheme();
+    const classes = useStyles(theme);
+    const dispatch = useDispatch();
+    const [passcode, setPasscode] = useState('');
+    const [success, setSuccess] = useState(false);
+    const [error, setError] = useState(false);
+    const [loading, setLoading] = useState(false);
+    const [blurred, setBlurred] = useState(false);
+    const maxLength = 6;
+    const inputId = 'passcode-input';
+
+    setTimeout(() => {
+        const input = document.getElementById(inputId);
+        if (input) {
+            input.focus();
+        }
+    });
```

Comment:
Should be able to just use the useEffect hook with no dependencies. It is called after the first render so the input element should be there by the time this is called. 
```suggestion
    useEffect(() => {
        const input = document.getElementById(inputId);
        if (input) {
            input.focus();
        }
    }, [])
```